ts/kpt-functions/src/types.ts

```diff
@@ -206,10 +210,46 @@ export class Configs {
     return cm ? cm.get(key) : undefined;
   }
 
+  /**
+   * Detects if an unknown value has been provided to the config map
+   * @param expectedKeys The set of expected keys for the function
+   * @returns Returns undefined if the config map is undefined. Otherwise
+   * returns a string[] containing the invalid keys. The string[] will be empty
+   * if all of the config maps keys are members of the expecteKeys. If the
+   * Configs expectedKeys is undefined this will return an empty [].
+   */
+  hasUnexpectedFunctionParameter(
```

Comment:
We don't need to expose this function, right?
And the comments above appear to be out-of-date

Reply:
nvm